include/dbus_utility.hpp

```diff
@@ -171,7 +174,7 @@ inline void getSubTreePaths(
         },
         "xyz.openbmc_project.ObjectMapper",
         "/xyz/openbmc_project/object_mapper",
-        "xyz.openbmc_project.ObjectMapper", "GetSubTreePaths", path, 0,
+        "xyz.openbmc_project.ObjectMapper", "GetSubTreePaths", path, depth,
         interfaces);
 }
```

Comment:
https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59428 seems expecting to have https://gerrit.openbmc.org/c/openbmc/sdbusplus/+/60020 too.

I'm not sure how extensive upstream refactored codes need to be adapted.
For example,

Upstream has 
 ```
   getDbusObject(const std::string& path,
                  std::span<const std::string_view> interfaces,
```
but downstream still uses this way.
`    getDbusObject(const std::string& path, std::span<const char*> interfaces,`

Reply:
I pulled only commits that are described in commit message. I don't need 60020 for fabricAdapter change.